com.musixmatch.Musixmatch.appdata.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.musixmatch.Musixmatch</id>
+  <launchable type="desktop-id">com.musixmatch.Musixmatch.desktop</launchable>
+  <name>Musixmatch</name>
+  <project_license>LicenseRef-proprietary=https://about.musixmatch.com/eula</project_license>
+  <developer_name>Musixmatch S.p.A.</developer_name>
+  <summary>Enjoy your lyrics everywhere</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://www.musixmatch.com/</url>
+  <url type="bugtracker">https://support.musixmatch.com</url>
+  <url type="help">https://support.musixmatch.com</url>
+  <description>
+    <p>
+      Enjoy lyrics everywhere - The world’s largest collection of song lyrics to enjoy with your music, on the go.
+    </p>
+    <p>
+      NOTE: This wrapper is not verified by, affiliated with, or supported by Musixmatch S.p.A.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>The main Musixmatch window</caption>
+      <image type="source" width="1140" height="720">https://raw.githubusercontent.com/flathub/com.musixmatch.Musixmatch/master/screenshots/main-view.png</image>
```

Comment:
The two screenshots url's are not working. I would suggest using your current branch for the screenshot url and update them once this get merged
```
• url-not-found         : <screenshot> failed to download (HTTP 404: Not Found) [https://raw.githubusercontent.com/flathub/com.musixmatch.Musixmatch/master/screenshots/main-view.png]
• url-not-found         : <screenshot> failed to download (HTTP 404: Not Found) [https://raw.githubusercontent.com/flathub/com.musixmatch.Musixmatch/master/screenshots/always-on-view.png]
```

Reply:
Thanks for your suggestions, it should be fixed.